package.json

```diff
@@ -27,7 +27,15 @@
     "broccoli-merge-trees": "^3.0.0",
     "broccoli-rollup": "^2.0.0",
     "broccoli-sass": "^0.7.0",
+    "eslint": "^4.19.1",
+    "eslint-config-standard": "^11.0.0",
+    "eslint-plugin-import": "^2.12.0",
+    "eslint-plugin-node": "^6.0.1",
+    "eslint-plugin-promise": "^3.7.0",
+    "eslint-plugin-standard": "^3.1.0",
     "rollup-plugin-alias": "^1.4.0"
   },
-  "dependencies": {}
-}
\ No newline at end of file
+  "dependencies": {
+    "broccoli-lint-eslint": "3.3.2"
```

Comment:
this is a devDependecy.

Reply:
moving this to a devDependency causes the following build error:

Error: standard:
	Configuration for rule "indent" is invalid:
	Value "data["1"].ImportDeclaration" has additional properties.
	Value "data["1"].flatTernaryExpressions" has additional properties.
	Value "data["1"].ignoreComments" has additional properties.

I suspect the reason is some conflict in the broccoli-lint-eslint.
When doing  npm ls --depth=9999 eslint and it's a dev dependency I get the following:
├─┬ broccoli-lint-eslint@3.3.2
│ └── eslint@3.19.0 
└── eslint@4.19.1 7

The same command whe the dependecy is just a dependency:

├─┬ broccoli-lint-eslint@3.3.2
│ └── UNMET DEPENDENCY eslint@^3.0.0
└── eslint@4.19.1 

It's a known issue that this happens when the latest version of the eslinter is not used 
https://github.com/standard/eslint-config-standard/issues/109

Any suggestions are welcome